aws/cloudformation/rdp_bastion.yaml

```
@@ -0,0 +1,501 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This template creates a bastion host based on Mate AMI on a new VPC
+  (customer).
+  This bastion host will be used to connect to the ROSA cluster, run oc commands
+  and view the OpenShift console.
+
+Parameters:
+  pTagprefix:
+    Description: Use the course code to prepend resource tags
+    Type: String
+    Default: cs220
```

```suggestion
    Default: cs220-cluster
```

Review the naming to fit the Herve's Cloudformation style

---

The customer vpc is outside the cs220-cluster (rosa) that's why I chose `cs220` and not `cs220-cluster`

---

@herve4m and @antmaro can I keep the tag prefix as cs220? else I'll have to update it everywhere in the commands while retrieving the IDs of the resources.

---

I'm keeping it as cs220 as it's easier to distinguish the resources of customer-vpc in AWS console